blocks/table/table.js

```diff
@@ -13,60 +13,76 @@ function buildCell(col, rowNum) {
   return cell;
 }
 
-async function buildDataTable(table, head, body, src) {
+async function buildDataTable(table, head, body, src, config) {
   const resp = await fetch(src);
   if (resp.ok) {
-    const json = await resp.json();
-    if (json.years && json.years[0] && json.years[0].tours && json.years[0].tours[0]) {
-      const { tournaments } = json.years[0].tours[0];
-      if (tournaments) {
-        const headRow = document.createElement('tr');
-        const headings = ['Year', 'Winner', 'Score', 'Runner Up', 'Score'];
```

Comment:
this is remnants of an old variant that is no longer used on the site, probably should have been removed a long time ago